updates/2016-07-17.yml

```diff
@@ -0,0 +1,9 @@
+changed:
+  -
+    title: "Update docs/articles/support/tickets.md"
+    tags:
+      - tickets
+      - support
+      - articles
+    description: |
+      Update the first section of the doc, forclarity and to better reflect the layout of the page.
```

Comment:
@mpaktiti Wow. There should be some sort of award for adding a typo to the phrase "for clarity". :)

Thanks for catching! Fixed and pushed.